packages/gestalt/dist/index.d.ts

```diff
@@ -1688,7 +1741,13 @@ interface SideNavigationNestedItemProps {
   label: string;
   active?: 'page' | 'section' | undefined;
   counter?: { number: string; accessibilityLabel: string } | undefined;
-  onClick?: ButtonEventHandlerType | undefined;
+  onClick?: AbstractEventHandler<
```

Comment:
still Button?

Reply:
SideNavigation component didn't change, so I decoupled it from Button type, that's why they're still the same type but no longer ButtonEventHandlerType (which is now only Button event)

Reply:
ok